src/assets/main.css

```diff
@@ -16,6 +16,8 @@
   --blue-france-850-200: var(--blue-cumulus-850-200);
   /* toastify */
   --toastify-color-success: #1f8d49; /* --success-main-525 (unavailable to date) */
+  /* DSFR contrast fix */
+  --blue-cumulus-sun-368-moon-732-hover--fix: oklch(57.03% 0.174 264.19);
```

Comment:
Je découvre `oklch` 👀 Et je ne comprends pas grand chose même en lisant la doc https://developer.mozilla.org/fr/docs/Web/CSS/color_value/oklch 😅 

Y'a-t-il une grosse plus-value par rapport à une valeur plus traditionnelle ?

Reply:
Il y a des avantages [expliqués ici](https://evilmartians.com/chronicles/oklch-in-css-why-quit-rgb-hsl). Je l'ai mis par réflexe mais je vais remettre du rgb/hex ça changera rien sur une seule valeur vu que c'est pas utilisé ailleurs.